### PR TITLE
Add splash screen

### PR DESCRIPTION
**Pull request by rubo**
- Added app loading spinner
- Added a notification when JavaScript is off
- Removed unused font download
- Moved Matomo script to `body`

### CODE REVIEW

**Review by Zishan-7 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -3,35 +3,99 @@
...
+        _paq.push(['setSiteId', '4']);
+        var d = document,
+          g = d.createElement('script'),
+          s = d.getElementsByTagName('script')[0];
+        g.type = 'text/javascript';
+        g.async = true;
+        g.src = u + 'matomo.js';
+        s.parentNode.insertBefore(g, s);
+      })();
+    </script>
```
Comment (vincenthongzy) [inline index.html:99]: oops my bad I wanted to put this before </body> but I ended up pasting it in before </html>

**Review by vincenthongzy — APPROVED** [review]
very nice spinner animation

**Comment by vincenthongzy** [comment]
`nit` but on smaller screens the text does not look as nice

image

**Comment by rubo** [comment]
> `nit` but on smaller screens the text does not look as nice

How did you achieve that? The text has padding and margin that works on different screen sizes like this:

image

**Comment by vincenthongzy** [comment]
image

I didn't do anything special though, its just chrome with a small window

**Comment by rubo** [comment]
> I didn't do anything special though, its just chrome with a small window

Turned out it was an extension issue.

**Pull request merged**